larpandoracontent/LArThreeDReco/LArTwoViewMatching/TwoViewSimpleTracksTool.cc

```diff
@@ -0,0 +1,127 @@
+/**
+ *  @file   larpandoracontent/LArThreeDReco/LArShowerMatching/TwoViewSimpleTracksTool.cc
+ *
+ *  @brief  Implementation of the clear showers tool class.
+ *
+ *  $Log: $
+ */
+
+#include "Pandora/AlgorithmHeaders.h"
+
+#include "larpandoracontent/LArThreeDReco/LArTwoViewMatching/TwoViewSimpleTracksTool.h"
+
+using namespace pandora;
+
+namespace lar_content
+{
+
+TwoViewSimpleTracksTool::TwoViewSimpleTracksTool() :
+    m_minMatchedFraction(0.2f),
+    m_minMatchingScore(0.9f),
+    m_minMatchedSamplingPoints(5),
+    m_minXOverlapFraction(0.5f)
+{
+}
+
+//------------------------------------------------------------------------------------------------------------------------------------------
+
+bool TwoViewSimpleTracksTool::Run(TwoViewTransverseTracksAlgorithm *const pAlgorithm, MatrixType &overlapMatrix)
+{
+    if (PandoraContentApi::GetSettings(*pAlgorithm)->ShouldDisplayAlgorithmInfo())
+       std::cout << "----> Running Algorithm Tool: " << this->GetInstanceName() << ", " << this->GetType() << std::endl;
+
+    ProtoParticleVector protoParticleVector;
+    this->FindBestTrack(overlapMatrix, protoParticleVector);
+
+    const bool particlesMade(pAlgorithm->CreateThreeDParticles(protoParticleVector));
+    return particlesMade;
```

Comment:
return `particlesMade(pAlgorithm->CreateThreeDParticles(protoParticleVector));`

Reply:
This seems to be the same point as in Leigh's comment above, will address both at once.